src/logic/calculate.js

```diff
@@ -60,12 +81,9 @@ const Calculate = (dataObj, btnName) => {
       next = null;
     }
   }
+  // convToString(total, next, operation)
```

Comment:
- [ ] [REQUIRED]  It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))